File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocsa/OCS0311Q00LayDownListInfo.java
package nta.med.data.model.ihis.ocsa;

public class OCS0311Q00LayDownListInfo {
	private String setPart          ;
	private String hangmogCode      ;
	private String setCode          ;
	private String hangmogCodeSet  ;
	private String hangmogName      ;
	private Double suryang           ;
	private String danui             ;
	private String danuiName        ;
	private String bulyongYn        ;
	private String slipName         ;
	private String inputControl     ;
	private String bunCode          ;
	public OCS0311Q00LayDownListInfo(String setPart, String hangmogCode,
			String setCode, String hangmogCodeSet, String hangmogName,
			Double suryang, String danui, String danuiName, String bulyongYn,
			String slipName, String inputControl, String bunCode) {
		super();
		this.setPart = setPart;
		this.hangmogCode = hangmogCode;
		this.setCode = setCode;
		this.hangmogCodeSet = hangmogCodeSet;
		this.hangmogName = hangmogName;
		this.suryang = suryang;
		this.danui = danui;
		this.danuiName = danuiName;
		this.bulyongYn = bulyongYn;
		this.slipName = slipName;
		this.inputControl = inputControl;
		this.bunCode = bunCode;
	}
	public String getSetPart() {
		return setPart;
	}
	public void setSetPart(String setPart) {
		this.setPart = setPart;
	}
	public String getHangmogCode() {
		return hangmogCode;
	}
	public void setHangmogCode(String hangmogCode) {
		this.hangmogCode = hangmogCode;
	}
	public String getSetCode() {
		return setCode;
	}
	public void setSetCode(String setCode) {
		this.setCode = setCode;
	}
	public String getHangmogCodeSet() {
		return hangmogCodeSet;
	}
	public void setHangmogCodeSet(String hangmogCodeSet) {
		this.hangmogCodeSet = hangmogCodeSet;
	}
	public String getHangmogName() {
		return hangmogName;
	}
	public void setHangmogName(String hangmogName) {
		this.hangmogName = hangmogName;
	}
	public Double getSuryang() {
		return suryang;
	}
	public void setSuryang(Double suryang) {
		this.suryang = suryang;
	}
	public String getDanui() {
		return danui;
	}
	public void setDanui(String danui) {
		this.danui = danui;
	}
	public String getDanuiName() {
		return danuiName;
	}
	public void setDanuiName(String danuiName) {
		this.danuiName = danuiName;
	}
	public String getBulyongYn() {
		return bulyongYn;
	}
	public void setBulyongYn(String bulyongYn) {
		this.bulyongYn = bulyongYn;
	}
	public String getSlipName() {
		return slipName;
	}
	public void setSlipName(String slipName) {
		this.slipName = slipName;
	}
	public String getInputControl() {
		return inputControl;
	}
	public void setInputControl(String inputControl) {
		this.inputControl = inputControl;
	}
	public String getBunCode() {
		return bunCode;
	}
	public void setBunCode(String bunCode) {
		this.bunCode = bunCode;
	}
	
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocsa/OCS0311Q00LayDownListInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocsa
{
    [Serializable]
    public class OCS0311Q00LayDownListInfo
    {
        private String _setPart;
        private String _hangmogCode;
        private String _setCode;
        private String _hangmogCodeSet;
        private String _hangmogName;
        private String _suryang;
        private String _danui;
        private String _danuiName;
        private String _bulyongYn;
        private String _slipName;
        private String _inputControl;
        private String _bunCode;

        public String SetPart
        {
            get { return this._setPart; }
            set { this._setPart = value; }
        }

        public String HangmogCode
        {
            get { return this._hangmogCode; }
            set { this._hangmogCode = value; }
        }

        public String SetCode
        {
            get { return this._setCode; }
            set { this._setCode = value; }
        }

        public String HangmogCodeSet
        {
            get { return this._hangmogCodeSet; }
            set { this._hangmogCodeSet = value; }
        }

        public String HangmogName
        {
            get { return this._hangmogName; }
            set { this._hangmogName = value; }
        }

        public String Suryang
        {
            get { return this._suryang; }
            set { this._suryang = value; }
        }

        public String Danui
        {
            get { return this._danui; }
            set { this._danui = value; }
        }

        public String DanuiName
        {
            get { return this._danuiName; }
            set { this._danuiName = value; }
        }

        public String BulyongYn
        {
            get { return this._bulyongYn; }
            set { this._bulyongYn = value; }
        }

        public String SlipName
        {
            get { return this._slipName; }
            set { this._slipName = value; }
        }

        public String InputControl
        {
            get { return this._inputControl; }
            set { this._inputControl = value; }
        }

        public String BunCode
        {
            get { return this._bunCode; }
            set { this._bunCode = value; }
        }

        public OCS0311Q00LayDownListInfo() { }

        public OCS0311Q00LayDownListInfo(String setPart, String hangmogCode, String setCode, String hangmogCodeSet, String hangmogName, String suryang, String danui, String danuiName, String bulyongYn, String slipName, String inputControl, String bunCode)
        {
            this._setPart = setPart;
            this._hangmogCode = hangmogCode;
            this._setCode = setCode;
            this._hangmogCodeSet = hangmogCodeSet;
            this._hangmogName = hangmogName;
            this._suryang = suryang;
            this._danui = danui;
            this._danuiName = danuiName;
            this._bulyongYn = bulyongYn;
            this._slipName = slipName;
            this._inputControl = inputControl;
            this._bunCode = bunCode;
        }

    }
}